modules/eightPodAnalyticsAdapter.js

```diff
@@ -176,24 +173,15 @@ eightPodAnalytics.eventsStorage = [];
 eightPodAnalytics.enableAnalytics = function (config) {
   eightPodAnalytics.originEnableAnalytics(config);
   logInfo(MODULE, 'init', config);
+  eightPodAnalytics.eventSubscribe();
 };
 
-eightPodAnalytics.disableAnalytics = (function (orig) {
-  return function (...args) {
-    if (interval) {
-      clearInterval(interval);
-      interval = null;
-    }
-    return orig.apply(this, args);
-  };
-})(eightPodAnalytics.disableAnalytics);
-
 /**
  * Register Analytics Adapter
  */
 adapterManager.registerAnalyticsAdapter({
   adapter: eightPodAnalytics,
-  code: MODULE_NAME
+  code: 'eightPod'
```

Comment:
on your ajax call, the keepalive method is much more polite; you may want to you it to get more events after the page dies.

Reply:
Thank you for great advice. I added keepalive.

Also I fixed my test, but they failed with another reason now